code/modules/clothing/suits/marine_armor.dm

```diff
@@ -730,6 +730,11 @@ var/list/squad_colors = list(rgb(230,25,25), rgb(255,195,45), rgb(200,100,200),
 	icon_state = "guardmedicarmor"
 	item_state = "guardmedicarmor"
 
+/obj/item/clothing/suit/storage/marine/imperial/sergeant/Initialize()
+	. = ..()
+	pockets.storage_slots = 3
```

Comment:
This is using the old method.

Reply:
this is actually a revert to what his original pr has, as above this does nothing because line 721 and 723-725

Reply:
Sorry, I fucked up resolving conflicts.